src/lib/modules/swarm/index.js

```diff
@@ -26,8 +26,15 @@ class Swarm {
     if(this.isSwarmEnabledInTheConfig() && cantDetermineUrl){
       console.warn('\n===== Swarm module will not be loaded =====');
       console.warn(`Swarm is enabled in the config, however the config is not setup to provide a URL for swarm and therefore the Swarm module will not be loaded. Please either change the ${'config/storage > upload'.bold} setting to Swarm or add the Swarm config to the ${'config/storage > dappConnection'.bold} array. Please see ${'https://embark.status.im/docs/storage_configuration.html'.underline} for more information.\n`);
-    }
-    if (!this.isSwarmEnabledInTheConfig() || cantDetermineUrl) {
+    } else {
```

Comment:
it seems you don't need an `else`

Reply:
I want to handle when it's not enabled on the config or poorly configured. How can I rewrite this without an `else`?

Reply:
Just return in the `if`

Reply:
Ignore my comment I thought it was the bottom of the function

Reply:
You could `return` before the end of the `if` block.